hw9/modules/player.ts

```diff
@@ -0,0 +1,90 @@
+import { Library } from "./library";
+import { Track } from "./track";
+import {EventEmitter} from 'events';
+export class Player {
+    public TracksPlayer: Array<Track>= [];
```

Comment:
the player should work with an array of libraries, since the library is an compilation of tracks

Reply:
fixed